tests/test_cli.py

```diff
@@ -595,3 +595,34 @@ def test_cli_with_python_mod():
 def test_doctor():
     res = CliRunner().invoke(doctor, catch_exceptions=False)
     assert res.exit_code == 0
+
+
+def test_mlflow_gc_with_datasets(sqlite_store):
+    store = sqlite_store[0]
+
+    mlflow.set_tracking_uri(sqlite_store[1])
+    mlflow.set_experiment("dataset")
+
+    dataset_source_url = (
```

Comment:
question: do we need to use the real dataset for this test? I wonder if this practice increases the text execution time.

Reply:
Oh true, we don't, let me update